tsup.config.ts

```diff
@@ -4,11 +4,11 @@ export default defineConfig([
   // src esm
   {
     entry: ['src/index.ts'],
-    outDir: 'dist/esm',
+    outDir: 'dist',
     clean: true,
     format: 'esm',
     target: ['es2022', 'node18'],
-    dts: false,
+    dts: true,
```

Comment:
**thought (blocking):** emitting cjs and esm types at the same time is not wanted right now
see my other comment